packages/cascara/src/modules/DataCheckbox/DataCheckbox.js

```diff
@@ -36,6 +36,7 @@ const DataCheckbox = ({
       <Checkbox
         {...rest}
         {...checkbox}
+        aria-label={label}
```

Comment:
We should _not_ be setting `aria-label` on any input that has an associated `<label>` tag that is linked with `htmlFor` to the `id` of the `<input>` itself.

The only time we would want to use `aria-label` on the input is in the rare scenario where we are not defining a label. We should be evaluating this conditionally if isLabeled is being set or not.

Reply:
@brianespinosa we have improved the way we are handling this, `aria-label` is only added when `isLabeled` is `false`.